### PR TITLE
Initial UI changes (search controls, result cards, styles, etc) for Urgent Care MVP part 1

### PR DESCRIPTION
**Pull request by brangi**
## Description
As a veteran, I need an updated UX experience so that I can easily locate a facility to meet my needs.
Original issue requirements:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/4872

## Testing done
 Functional testing done with mock data but also with facilities api locally.

## Screenshots/
Screen Shot 2020-01-24 at 6 36 26 PM

Screen Shot 2020-01-24 at 7 02 37 PM

Screen Shot 2020-01-24 at 7 02 52 PM

## Acceptance criteria
- [ ] - department-of-veterans-affairs/va.gov-team#4872

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by darkbushido — APPROVED** [review]
LGTM

File: src/applications/facility-locator/constants/mock-facility-data.json.org [hunk 1]
```diff
@@ -0,0 +1,1031 @@
+{
```
Comment (cvalarida) [inline src/applications/facility-locator/constants/mock-facility-data.json.org]: Is the `.org` extension intentional?

**Review by cvalarida — APPROVED** [review]
Gonna be honest: I barely looked at the code. :see_no_evil:
CodeClimate says it's got 100% test coverage, though, so :shipit:

File: src/applications/facility-locator/constants/mock-facility-data.json.org (hunk 1 above)
Comment (brangi) [reply src/applications/facility-locator/constants/mock-facility-data.json.org]: @cvalarida thanks for approving this, the file is used to testing. I use it at the moment for mock data. Will remove later.

**Pull request merged**